ci-operator/jobs/openshift/machine-api-operator/openshift-machine-api-operator-master-presubmits.yaml

```diff
@@ -212,6 +212,7 @@ presubmits:
       ci-operator.openshift.io/prowgen-controlled: "true"
       pj-rehearse.openshift.io/can-be-rehearsed: "true"
     name: pull-ci-openshift-machine-api-operator-master-e2e-aws-upgrade
+    optional: true
```

Comment:
I think you've added this to the wrong one, this is for the AWS upgrade right?

Reply:
good catch, fixed